tests/integration/test_cloudformation.py

```diff
@@ -1724,15 +1724,40 @@ def test_delete_stack(self):
         cloudformation.delete_stack(StackName='myteststack2')
         cloudformation.delete_stack(StackName='myteststack')
 
-    def test_delete_stack_across_regions(self):
-        domain_name = 'es-%s' % short_uid()
-
-        cloudformation = aws_stack.connect_to_service('cloudformation', region_name='eu-central-1')
+    def test_globalindex_read_write_provisioned_throughput_dynamodb_table(self):
```

Comment:
nit: Can we preserve the previous test functionality (deleting stacks across regions) here and leave the `region_name` parameter in the client creation?